lib/concurrent-ruby/concurrent/configuration.rb

```diff
@@ -149,7 +170,6 @@ def self.executor(executor_identifier)
   def self.new_fast_executor(opts = {})
     FixedThreadPool.new(
         [2, Concurrent.processor_count].max,
-        auto_terminate:  opts.fetch(:auto_terminate, true),
```

Comment:
@pitr-ch don't you want to pass down the `auto_terminate` option?
I guess someone _may_ want one non-daemonized

Reply:
Indeed. I missed it does not merge with the opts. Good catch.